src/adaptors/http1/http1_server.c

```diff
@@ -143,11 +143,10 @@ static void _send_request_message(_server_request_t *hreq);
 // An HttpConnector has been created.  Create an qdr_http_connection_t and a
 // qdr_connection_t for it.
 //
-static qdr_http1_connection_t *_create_server_connection(qd_http_connector_t *ctor,
-                                                         qd_dispatch_t *qd,
-                                                         const qd_http_adaptor_config_t *config)
+static qdr_http1_connection_t *_create_server_connection(qd_http_connector_t *ctor, qd_dispatch_t *qd)
```

Comment:
This is minor but can we call this `qd_http_connector_t *` parameter not `ctor` but `connector` instead ?

Reply:
Yes that's a good idea